src/main/java/com/alkemy/ong/dto/CreateNewsDto.java

```diff
@@ -0,0 +1,24 @@
+package com.alkemy.ong.dto;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import com.alkemy.ong.model.Category;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateNewsDto {
+
+	@NotBlank
+	private String name;
+	@NotBlank
+	private String content;
+	@NotBlank
+	private String image;
+	
+	@NotNull
+	private Category category;
```

Comment:
Aca deberia ser un `Dto`. (Se deberia cambiar en cascada en los lugares que se requiera)

```suggestion
	private CategoryDto category;
```
**o** 

directamente utilizar un string para el nombre de categoria. y buscarlo por nombre, agregarlo con un setter a la category que se busque, y tambien comprobar si se encuentra... 

```suggestion
	private String categoryName;
```

Aunque no se bien como es el ticket. Solo es una sugerencia.